frontend/src/scenes/settings/SettingsScene.stories.tsx

```diff
@@ -0,0 +1,215 @@
+
+export const SettingsUserApiKeys: Story = Template.bind({})
+SettingsUserApiKeys.args = { sectionId: 'user-api-keys' }
+
+export const SettingsUserNotifications: Story = Template.bind({})
+SettingsUserNotifications.args = { sectionId: 'user-notifications' }
+
+export const SettingsUserCustomization: Story = Template.bind({})
+SettingsUserCustomization.args = { sectionId: 'user-customization' }
+
+export const SettingsUserDangerZone: Story = Template.bind({})
+SettingsUserDangerZone.args = { sectionId: 'user-danger-zone' }
+
+// NOTE: This is used to guarantee we're testing all sections
+const _ALL_SECTIONS_CHECKER: Record<SettingSectionId, Story> = {
+    'environment-details': SettingsEnvironmentDetails,
+    'environment-autocapture': SettingsEnvironmentAutocapture,
+    'environment-product-analytics': SettingsEnvironmentProductAnalytics,
+    'environment-revenue-analytics': SettingsEnvironmentRevenueAnalytics,
+    'environment-marketing-analytics': SettingsEnvironmentMarketingAnalytics,
+    'environment-web-analytics': SettingsEnvironmentWebAnalytics,
+    'environment-replay': SettingsEnvironmentReplay,
+    'environment-surveys': SettingsEnvironmentSurveys,
+    'environment-feature-flags': SettingsEnvironmentFeatureFlags,
+    'environment-error-tracking': SettingsEnvironmentErrorTracking,
+    'environment-csp-reporting': SettingsEnvironmentCSPReporting,
+    'environment-max': SettingsEnvironmentMax,
+    'environment-integrations': SettingsEnvironmentIntegrations,
+    'environment-access-control': SettingsEnvironmentAccessControl,
+    'environment-danger-zone': SettingsEnvironmentDangerZone,
+    'project-details': SettingsProjectDetails,
+    'project-danger-zone': SettingsProjectDangerZone,
+    'project-autocapture': SettingsProjectAutocapture,
+    'project-product-analytics': SettingsProjectProductAnalytics,
+    'project-replay': SettingsProjectReplay,
+    'project-surveys': SettingsProjectSurveys,
+    'project-integrations': SettingsProjectIntegrations,
+    'project-access-control': SettingsProjectAccessControl,
+    'organization-details': SettingsOrganizationDetails,
+    'organization-members': SettingsOrganizationMembers,
+    'organization-roles': SettingsOrganizationRoles,
+    'organization-authentication': SettingsOrganizationAuthentication,
+    'organization-proxy': SettingsOrganizationProxy,
+    'organization-danger-zone': SettingsOrganizationDangerZone,
+    'organization-billing': SettingsOrganizationBilling,
+    'organization-startup-program': SettingsOrganizationStartupProgram,
+    'user-profile': SettingsUserProfile,
+    'user-api-keys': SettingsUserApiKeys,
+    'user-notifications': SettingsUserNotifications,
+    'user-customization': SettingsUserCustomization,
+    'user-danger-zone': SettingsUserDangerZone,
+}
+
+// NOTE: This is here to avoid TS complaining
+// about us not using the variable used above to check all sections
+;(() => _ALL_SECTIONS_CHECKER)()
```

Comment:
Why not use a `// @ts-expect-error` comment instead?

Reply:
Because I wanna check with Typescript that we have a snapshot for every possible setting. If I disable Typescript to avoid the "umused variable" warning, it also defeats the purpose of using Typescript to check for full coverage.

I would love a way to disable a single TS rule rather than all or nothing